### ISSUE TITLE
Zig master compile error

### ISSUE THREAD

**Issue by VisenDev** [capy-ui/capy#73]
The repo fails to build with the latest zig master

```
Robert@Roberts-MacBook-Pro-2 ~/z/capy (master)> zig build
/Users/Robert/zig/capy/build.zig:14:34: error: no field or member function named 'getOsTag' in 'Build.ResolvedTarget'
    const broken = switch (target.getOsTag()) {
                           ~~~~~~^~~~~~~~~
/Users/Robert/zig/zig/build/stage3/lib/zig/std/Build.zig:2276:28: note: struct declared here
pub const ResolvedTarget = struct {
```

**Comment by zenith391** [comment]
fixed by 4d1778feba39949fb2569379b8ac4aabf4842478 although capy now targets Zig version `0.12.0-dev.2063+804cee3b9` instead of `master`.

**Comment by antomfdez** [comment]
![image](https://github.com/user-attachments/assets/20f9aea6-84ab-4acd-8877-e8dbd9e5cf02)
error compiling in dev version of zig
![image](https://github.com/user-attachments/assets/a7e52da8-72ea-4c3b-af89-c897135cddce)
